pkg/yaml2pcl/yaml2pcl.go

```diff
@@ -191,6 +191,21 @@ func walkToPCL(v Visitor, node ast.Node, totalPCL io.Writer, suffix string) erro
 
 	tk := node.GetToken()
 	switch n := node.(type) {
+	case *ast.LiteralNode:
```

Comment:
Does this handle other sorts of YAML literals (`>` `+`, etc)? https://yaml-multiline.info/

We should definitely add a couple of tests that exercise more of the yaml multiline string spec.

Reply:
ast.LiteralNode registers all other literals properly, however does not format them in the intended way. Will take care of this in a separate PR